usaspending_api/api_contracts/contracts/v2/agency/toptier_code/recipients.md

```diff
@@ -0,0 +1,50 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Recipients [/api/v2/agency/{toptier_code}/recipients/{?fiscal_year}]
+
+Returns a list of data points of an agencies recipients for a given fiscal year
+
+## GET
+
++ Request (application/json)
```

Comment:
Lines  10-16 are unnecessary for a GET request and prevent the mock API from working
```
[MATCHING] by url pattern: /api/v2/agency/:toptier_code/recipients/ MATCHED
[MATCHING] by request content type: application/json actual:  NOT_MATCHED
```

Reply:
Okay, should be good to go.